templates/part.content.php

```diff
@@ -88,13 +89,90 @@ class="icon-toggle keep-unread"
                         </div>
                     </li>
                 </ul>
+                <!-- Share dropdown -->
+                <ng-container click-outside="Content.hide()" news-stop-propagation>
+                    <div
+                        ng-controller="ShareController as Share"
+                        ng-if="Content.showDropdown[item.id]"
+                        class="dropdown-content"
+                        news-stop-propagation>
+                        <!-- Share with users -->
+                        <p class="label-group"><?php p($l->t('Share with users')) ?></p>
+                        <form ng-submit=""
+                            name="contactForm"
+                            autocomplete="off">
+                            <fieldset class="contact-input">
+                                <input
+                                    ng-model="nameQuery"
+                                    ng-model-options="{debounce: 400}"
+                                    ng-change="Share.searchUsers(nameQuery)"
+                                    type="text"
+                                    placeholder="<?php p($l->t('Username')) ?>"
+                                    title="<?php p($l->t('Username')) ?>"
+                                    name="contactName"
+                                    required
+                                    style="width: 200px">
+                                    <div ng-if="App.loading.isLoading('user')"
+                                        ng-class="{'icon-loading-small': App.loading.isLoading('user') }">
+                                    </div>
+                            </fieldset>
+                        </form>
+
+                        <div style="margin-left: 2em; font-size: 0.85em;"
+                            ng-if="Share.userList.length === 0 && nameQuery && !App.loading.isLoading('user') && !Share.searchUsersFailed">
+                            <?php p($l->t('No users found')) ?>
+                        </div>
+                        <div style="margin-left: 2em; font-size: 0.85em;"
+                            ng-if="Share.userList.length === 0 && Share.searchUsersFailed">
+                            <?php p($l->t('Error while searching for users')) ?>
+                        </div>
+                        <a
+                            ng-repeat="user in Share.userList"
+                            class="icon-category-installed pr-3"
+                            ng-click="Share.shareItem(item.id, user.value.shareWith)">
+                            {{ user.label }}
+                            <span class="right" style="margin-top: 1.1em; margin-right: 1em"
+                                ng-class="{'icon-loading-small': Share.isLoading(user.value.shareWith), 'icon-checkmark': Share.isStatus(item.id, user.value.shareWith, true), 'icon-close': Share.isStatus(item.id, user.value.shareWith, false)}">
+                            </span>
```

Comment:
Why do you use a dedicated `right` class just for this single use of `float: right;` but add the margins entirely inline?